diff --git a/budgie-visualspace/data/meson.build b/budgie-visualspace/data/meson.build
index bc3734a..8e316be 100644
--- a/budgie-visualspace/data/meson.build
+++ b/budgie-visualspace/data/meson.build
@@ -4,7 +4,7 @@ install_data(
     install_dir: PIXMAPS_DIR
 )
 
-AUTOSTART_LOC=join_paths(sysconfdir, 'xdg', 'autostart')
+AUTOSTART_LOC=join_paths(datadir, 'xdg', 'autostart')
 substprog = find_program('subst.py')
 
 mytarget = custom_target('autoworkspace',
-- 
2.26.0

