diff --git a/budgie-wallstreet/src/wallstreet_control/wallstreet_control.vala b/budgie-wallstreet/src/wallstreet_control/wallstreet_control.vala
index a1a4a91..ae99398 100644
--- a/budgie-wallstreet/src/wallstreet_control/wallstreet_control.vala
+++ b/budgie-wallstreet/src/wallstreet_control/wallstreet_control.vala
@@ -206,7 +206,7 @@ namespace WallStreetControls {
         }
 
         private bool processruns (string application) {
-            string cmd = "/usr/bin/pgrep -f " + application;
+            string cmd = "/bin/pgrep -f " + application;
             string output;
             try {
                 GLib.Process.spawn_command_line_sync(cmd, out output);
@@ -316,4 +316,4 @@ namespace WallStreetControls {
         Gtk.main();
         return 0;
     }
-}
\ No newline at end of file
+}
-- 
2.26.0

