diff --git a/budgie-quickchar/quickchar/data/meson.build b/budgie-quickchar/quickchar/data/meson.build
index 76cada6..452a552 100644
--- a/budgie-quickchar/quickchar/data/meson.build
+++ b/budgie-quickchar/quickchar/data/meson.build
@@ -6,7 +6,7 @@ install_data(
 
 install_data(
     'quickchar-autostart.desktop',
-    install_dir: join_paths(sysconfdir, 'xdg', 'autostart')
+    install_dir: join_paths(datadir, 'xdg', 'autostart')
 )
 
 install_data(
-- 
2.26.0

