diff --git a/budgie-weathershow/src/weathershow/WeatherShow.vala b/budgie-weathershow/src/weathershow/WeatherShow.vala
index 88814fe..7bfb4cf 100644
--- a/budgie-weathershow/src/weathershow/WeatherShow.vala
+++ b/budgie-weathershow/src/weathershow/WeatherShow.vala
@@ -77,7 +77,7 @@ namespace WeatherShowFunctions {
     }
 
     private bool check_onwindow(string path) {
-        string cmd_check = "/usr/bin/pgrep -f " + path;
+        string cmd_check = "/bin/pgrep -f " + path;
         string output;
         try {
             GLib.Process.spawn_command_line_sync(cmd_check, out output);
@@ -97,7 +97,7 @@ namespace WeatherShowFunctions {
         if (win_exists) {
             try {
                 Process.spawn_command_line_async(
-                    "/usr/bin/pkill -f ".concat(path));
+                    "/bin/pkill -f ".concat(path));
             }
             catch (SpawnError e) {
                 /* nothing to be done */
-- 
2.26.0

