diff --git a/Ventoy2Disk.sh b/Ventoy2Disk.sh
index dac5c2f..68e6f2a 100755
--- a/Ventoy2Disk.sh
+++ b/Ventoy2Disk.sh
@@ -42,11 +42,12 @@ if ! [ -f ./boot/boot.img ]; then
     exit 1
 fi
 
-echo "############# Ventoy2Disk $* [$TOOLDIR] ################" >> ./log.txt
-date >> ./log.txt
+# save logs in /var/log/ventoy
+echo "############# Ventoy2Disk $* [$TOOLDIR] ################" >> /var/log/ventoy/log.txt
+date >> /var/log/ventoy/log.txt
 
 #decompress tool
-echo "decompress tools" >> ./log.txt
+echo "decompress tools" >> /var/log/ventoy/log.txt
 cd ./tool/$TOOLDIR
 
 ls *.xz > /dev/null 2>&1
@@ -54,7 +56,7 @@ if [ $? -eq 0 ]; then
     [ -f ./xzcat ] && chmod +x ./xzcat
 
     for file in $(ls *.xz); do
-        echo "decompress $file" >> ./log.txt
+        echo "decompress $file" >> /var/log/ventoy/log.txt
         xzcat $file > ${file%.xz}
         [ -f ./${file%.xz} ] && chmod +x ./${file%.xz}
         [ -f ./$file ] && rm -f ./$file
-- 
2.34.0

