diff --git a/usr/share/ventoy/Ventoy2Disk.sh b/usr/share/ventoy/Ventoy2Disk.sh
index d8440d0..cd526f3 100755
--- a/Ventoy2Disk.sh
+++ b/Ventoy2Disk.sh
@@ -42,12 +42,14 @@ if ! [ -f ./boot/boot.img ]; then
     exit 1
 fi
 
-echo "############# Ventoy2Disk $* [$TOOLDIR] ################" >> ./log.txt
-date >> ./log.txt
+# create log dir in $HOME
+mkdir -p $HOME/.ventoy
+echo "############# Ventoy2Disk $* [$TOOLDIR] ################" >> $HOME/.ventoy/log.txt
+date >> $HOME/.ventoy/log.txt
 
 #decompress tool
 if [ -f ./tool/$TOOLDIR/ash ]; then
-    echo "no need to decompress tools" >> ./log.txt
+    echo "no need to decompress tools" >> $HOME/.ventoy/log.txt
 else
     cd ./tool/$TOOLDIR
     
-- 
2.32.0

